### PR TITLE
docs: add page for in app browser plugin

### PR DESCRIPTION
**Pull request by ItsChaceD**
This PR adds the page for the `InAppBrowser` plugin documentation

### CODE REVIEW

**Review by OS-ricardomoreirasilva — APPROVED** [review]

File: docs/apis/inappbrowser.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+title: InAppBrowser Capacitor Plugin API
+description: The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security.
+custom_edit_url: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/README.md
+editApiUrl: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/src/definitions.ts
+sidebar_label: InAppBrowser
+---
+
+# @capacitor/inappbrowser
+
+The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security.
```
Comment (alexgerardojacinto) [inline docs/apis/inappbrowser.md:11]: Should we mention the 3 different ways that the plugin offers to open URLs? That is, in a WebView, in an in-app system browser (Custom Tabs and Safari), and in the device's default browser.

File: docs/apis/inappbrowser.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+```typescript
+removeAllListeners() => void
+```
+
+Removes all listeners for the browser events.
+
+--------------------
+
+
+### addListener('browserClosed' | 'browserPageLoaded', ...)
```
Comment (alexgerardojacinto) [inline docs/apis/inappbrowser.md:142]: Should `addListener` come before `removeAllListeners`?

File: docs/apis/inappbrowser.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+| **`bottomSheetOptions`**  | <code><a href="#androidbottomsheet">AndroidBottomSheet</a></code> | Sets the options for the bottom sheet when this is selected as the viewStyle. If viewStyle is FULL_SCREEN, this will be ignored. |
+| **`startAnimation`**      | <code><a href="#androidanimation">AndroidAnimation</a></code>     | Sets the start animation for when the browser appears.                                                                           |
+| **`exitAnimation`**       | <code><a href="#androidanimation">AndroidAnimation</a></code>     | Sets the exit animation for when the browser disappears.                                                                         |
+
+
+#### AndroidBottomSheet
+
+| Prop          | Type                 | Description                             |
+| ------------- | -------------------- | --------------------------------------- |
+| **`height`**  | <code>number</code>  | Sets the height of the bottom sheet.    |
```
Comment (alexgerardojacinto) [inline docs/apis/inappbrowser.md:244]: Should we state that if no value is passed, then `height` will be 50% of the screen's height?

**Comment by Annie602942** [comment]
https://github.com/vikramlearning/blazorbootstrap/pull/806

File: docs/apis/inappbrowser.md (hunk 1 above)
Comment (dallastjames) [reply docs/apis/inappbrowser.md:11]: I agree. I think we should include this information.

File: docs/apis/inappbrowser.md [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+                password = ""
+            }
+            content {
+                includeGroup "com.github.outsystems"
+            }
+        }
+    }
+}
+```
+
```
Comment (dallastjames) [inline docs/apis/inappbrowser.md:56]: Can we add an example section here showing how to use this in an application? I'm sure it's really simple, but it's some feedback I hear quite often about the plugin docs that they lack examples.

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
plugin docs are generated from the plugin README.md file (and parts of it are generated from the definitions.ts file)

This file content doesn't match the plugins README.md file. All the information should be added [there](https://github.com/ionic-team/capacitor-os-inappbrowser/blame/dev/README.md) 

Then, after a plugin release the plugin docs generation script can be run to pick the changes.
All the requested changes should be applied on the plugin repository, not here.

**Review by Annie602942 — APPROVED** [review]

**Comment by Annie602942** [comment]
https://github.com/Annie602942/good.u.ANN/issues/1

File: docs/apis/inappbrowser.md [hunk 5]
```diff
@@ -0,0 +1,381 @@
+---
+title: InAppBrowser Capacitor Plugin API
+description: The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security. It offers three different ways to open URLs: in a WebView, in an in-app system browser (Custom Tabs for Android and SFSafariViewController for iOS), and in the device's default browser.
+custom_edit_url: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/README.md
+editApiUrl: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/src/definitions.ts
+sidebar_label: InAppBrowser
+---
+
+# @capacitor/inappbrowser
```
Comment (Annie602942) [inline docs/apis/inappbrowser.md:9]: https://github.com/Annie602942/good.u.ANN/issues/1

Comment (Annie602942) [reply docs/apis/inappbrowser.md:9]: docs/apis/inappbrowser.md

**Review by Annie602942 — COMMENTED** [review]
docs/apis/inappbrowser.md

**Review by Annie602942 — APPROVED** [review]

**Review by IT-MikeS — APPROVED** [review]

**Pull request merged**